backend/src/main/java/instantPolls/model/Room.java

```diff
@@ -2,23 +2,69 @@
 
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
 import java.util.TimeZone;
+import java.util.stream.Collectors;
 
 public class Room {
 	
 	private String id;
+	private String shortId;
 	private String token;
 	private String roomName;
 	private TimeZone timeZone;
 	private LocalDate expirationDate;
-	private int numberOfUsers;
+	private ArrayList<Question> questions;
+	private List<String> users;
+	private int iterator;
 	
 	public Room(String id, String roomName, LocalDate expirationDate, String timeZoneName) {
 		this.id = id;
 		this.roomName = roomName;
 		this.timeZone = TimeZone.getTimeZone(timeZoneName);
 		this.expirationDate = expirationDate;
 		expirationDate.format(DateTimeFormatter.ofPattern("dd.MM.yyyy"));
+		questions = new ArrayList<Question>();
+		users = new ArrayList<>();
+	}
+	
+	public ArrayList<HashMap<String,Object>> getListOfQuestionsWithVotes(String userId) {

```

Comment:
Return types should be as generic as possible. This method shoud return just List<Map<String, Object>> Noone cares about specific impelementation of list used here. Also Object here is a little bit disturbing.